app/src/controllers/chefs.ts

```diff
@@ -52,17 +51,15 @@ const controller = {
 
   getSubmission: async (req: Request, res: Response, next: NextFunction) => {
     try {
-      const response = await chefsService.getSubmission(req.query.formId as string, req.params.submissionId);
-      res.status(200).send(response);
+      res.status(200).send(await chefsService.getSubmission(req.query.formId as string, req.params.submissionId));
```

Comment:
- `req.query.formId` should be getting pre-processed by the `addDashesToUuid` utility instead of being implicit casted into a string. Eventual Joi validation layer should be catching whether it's a valid uuidv4 at all or not down the line but for now need to make sure we're not just accepting potentially malicious strings as inputs and having it go straight into the service layer.
- I have mixed feelings about in-lining the response into the express send argument, as we generally have kept them separated out due to the need for it to async/await on a service call. While it is semantically the same, in the event there is an error, it becomes slightly harder to directly track which line triggered the fault in the stacktrace if it's all jumbled together in one line. Personally leaning towards recommending we keep it const response/send(response) style even though it's slightly more lines to keep things consistent with other controllers.